tests/test_as_readme_text.py

```diff
@@ -52,3 +54,53 @@ def test_creates_readme():
 def test_creates_readme_with_empty_values():
     """Should be able to create a README for an empty set of properties."""
     assert as_readme_text(PackageProperties())
+
+
+@mark.parametrize(
+    "description, expected",
+    [
+        # Empty or whitespace-only.
+        ("", ""),
+        ("    ", ""),
+        # Single line text.
+        ("Non-indented one-line text.", "Non-indented one-line text."),
+        ("    Indented one-line text.", "Indented one-line text."),
+        (
+            "    Indented one-line text with trailing whitespace.   ",
+            "Indented one-line text with trailing whitespace.",
```

Comment:
```suggestion
            "Indented one-line text with trailing whitespace.   ",
```
To not imply that it removes trailing whitespace